include/sawtooth/transaction_processor.h

```diff
@@ -24,6 +24,30 @@
 
 namespace sawtooth {
 
+// This is the version used by SDK to match if validator supports feature
+// it requested during registration. It should only be incremented when
+// there are changes in TpRegisterRequest. Remember to sync this
+// information in validator if changed.
+// Note: SdkProtocolVersion is the highest version the SDK supports
+enum class FeatureVersion: uint32_t {
+    FeatureUnused = 0,
+    SdkProtocolVersion = 0,
+};
+
+static const unsigned int FeatureVersionToUnsignedInt(
+        const FeatureVersion& feature_version) {
+    unsigned int value;
+    switch (feature_version) {
+        case FeatureVersion::FeatureUnused:
+            value = 0;
+            break;
+        // case FeatureVersion::SdkProtocolVersion:
```

Comment:
Is this supposed to be a comment? This looks like commented out code

Reply:
Yes, it is a comment. I made it code style comment instead of explaining in words. This case is already covered.